docs/sources/mimir/migration-guide/migrate-from-cortex.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Migrate from Cortex to Grafana Mimir"
+menuTitle: "Migrate from Cortex"
+description: "Learn how to migrate your deployment of Cortex to Grafana Mimir to simplify the deployment and continued operation of a horizontally scalable, multi-tenant time series database with long-term storage."
+weight: 10
+---
+
+# Migrate from Cortex to Grafana Mimir
+
+This document guides an operator through the process of migrating a Jsonnet deployment of [Cortex](https://cortexmetrics.io/) to Grafana Mimir.
+This document includes an overview of the steps required for any environment, and specific instructions for [environments deployed with Jsonnet](#migrating-to-grafana-mimir-using-jsonnet).
+This document was tested with Cortex versions 1.10 and 1.11. While it may work with more recent versions of Cortex, we cannot guarantee that.
+To migrate a Helm deployment of Cortex refer to [Migrate from Cortex](/docs/helm-charts/mimir-distributed/latest/migration-guides/migrate-from-cortex).
+
+Grafana Mimir includes significant changes that simplify the deployment and continued operation of a horizontally scalable, multi-tenant time series database with long-term storage.
+
+The changes focus on making Grafana Mimir easier to run out of the box, including:
+
+- Removing configuration parameters that don't require tuning
+- Renaming some parameters so that they're more easily understood
+- Updating the default values of some existing parameters
+
+The `mimirtool` automates configuration conversion.
+It provides a simple migration by generating Mimir configuration from Cortex configuration.
+
+## Before you begin
```

Comment:
```suggestion
**Before you begin:**
```
Style change since the previous version of this doc was written; see https://github.com/grafana/writers-toolkit/edit/main/docs/sources/writing-guide/topic-types/task/index.md.